component/container-runtime.jsonnet

```
@@ -6,15 +6,38 @@ local inv = kap.inventory();
 
 local params = inv.parameters.openshift4_nodes;
 
+local mergedConfigs =
+  std.foldl(
+    function(configs, name)
+      local pool = params.machineConfigPools[name];
+      configs {
+        [if std.objectHas(pool, 'containerRuntime') then name]: pool.containerRuntime,
+      },
+    std.objectFields(params.machineConfigPools),
+    {}
+  ) + com.makeMergeable(params.containerRuntimeConfigs);
+
```

This could probably be a helper which is shared between container-runtime and kubelet configs.

```suggestion
local mergedConfigs(configtype) =
  std.foldl(
    function(configs, name)
      local pool = params.machineConfigPools[name];
      configs {
        [if std.objectHas(pool, configtype) then name]: pool[configtype],
      },
    std.objectFields(params.machineConfigPools),
    {}
  ) + com.makeMergeable(params["%sConfigs" % configtype]);

```